asapdiscovery-docking/asapdiscovery/docking/scorer.py

```diff
@@ -394,21 +663,24 @@ def get_ml_scorer_cls_from_model_type(model_type: ModelType):
 
 class MetaScorer(BaseModel):
     """
-    Score from a combination of other scorers
+    Score from a combination of other scorers, the scorers must share an input type,
     """
 
     scorers: list[ScorerBase] = Field(..., description="Scorers to score with")
 
     def score(
         self,
-        inputs: list[DockingResult],
+        inputs: list[Any],
```

Comment:
Should this be a union of the supported types list[DockingResult | Complex | Path] ?

Reply:
Yes good catch, I was lazy.